config/routes.rb

```diff
@@ -147,6 +157,7 @@
   end
 
   get '/admin' => redirect('/admin/conferences')
-
+  conf = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc).first
+  get '/' => redirect("/conferences/#{conf.short_title}"), constraints: RootConstraint.new
```

Comment:
Why do you create the root different here?
Does 
```
    root to: redirect("/conferences/#{conf.short_title}"), constraints: RootConstraint.new
```
not work? (or something similar...)